build-11.gradle

```diff
@@ -123,6 +123,11 @@ task integrationTest(type: Test) {
         systemProperty "testprofile", "integration"
     }
 }
+task ScaIntegrationTest(type: Test) {
+    useJUnitPlatform() {
+        systemProperty "testprofile", "scaIntegration"
+    }
+}
 
```

Comment:
don't we need all the sections we added to the java8 build?

Reply:
no, because no one runs the tasks on the other build.gradle except the main one
I will remove this one also from here